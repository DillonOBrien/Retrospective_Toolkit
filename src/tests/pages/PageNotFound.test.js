import React from "react";
import { shallow } from "enzyme";
import PageNotFound from "../../pages/PageNotFound";

let wrapper;

beforeEach(() => {
  wrapper = shallow(<PageNotFound />);
});

describe("Pages/PageNotFound Testing", () => {
  // Snapshot for PageNotFound
  test("SNAPSHOT: PageNotFound", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
