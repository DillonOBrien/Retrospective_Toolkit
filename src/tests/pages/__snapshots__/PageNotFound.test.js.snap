// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages/PageNotFound Testing SNAPSHOT: PageNotFound 1`] = `
<div
  className="makeStyles-root-1"
>
  <img
    alt="Oops! Page Not Found"
    className="makeStyles-img-2"
    src="https://d1g31diwtzkeb3.cloudfront.net/404.gif"
  />
  <h1
    className="makeStyles-header-3"
  >
    404! Oops...
  </h1>
  <div
    className="makeStyles-body-4"
  >
    <h2>
      The page you are looking for doesn't exist!
    </h2>
    <WithStyles(ForwardRef(Button))
      className="makeStyles-button-6"
    >
      <Link
        className="makeStyles-goHome-5"
        to="/"
      >
        Go Home
      </Link>
    </WithStyles(ForwardRef(Button))>
  </div>
</div>
`;
